lisa/mixin_modules.py

```diff
@@ -18,6 +18,7 @@
 import lisa.notifiers.perfdump  # noqa: F401
 import lisa.notifiers.text_result  # noqa: F401
 import lisa.runners.lisa_runner  # noqa: F401
+import lisa.sut_orchestrator.azure.notifiers  # noqa: F401
```

Comment:
Move this line into L30, Azure modules, so it won't have error, when Azure is not installed.

Reply:
Done